about.html

```diff
@@ -0,0 +1,171 @@
+          </div>
+          <div class="time_place-abt">
+            <p>
+              Please contact on our website form for any further questions about
+              The NAMM Show East Africa 2023!
+            </p>
+            <br />
+            <h4>
+              <a href="https://www.namm.org/contact"
+                >https://www.namm.org/contact</a
+              >
+            </h4>
+          </div>
+        </div>
+      </section>
+      </div>
+      <section class="logo">
+        <h2>The Namm Show 2023 Logo</h2>
+        <span class="line"></span>
+        <p>
+          The logo of The Namm Show 2023 was decided through the logo
+          competition from 8th February to 7th March. <br />
+          <br />
+          <b class="bold_name">"Jim Cruickshank"</b> won the competition. He had
+          an eye for design! While a proud member of the Fender guitar team, Jim
+          designed many of the most memorable trade show displays for the NAMM
+          company.
+        </p>
+        <img src="./assets/NAMMlogo-2.jpg" alt="The NAMM Logo" />
+      </section>
+      <section class="previous_sessions">
+        <h2>See the past NAMM Shows</h2>
+        <span class="line"></span>
+        <p>
+          Take a look at the last two NAMM Shows which took place in Anaheim,
+          California, US at the Anaheim Convention Center.
+        </p>
+
+        <div class="session_photos">
+          <div class="First_photo pic">
+            <h2>2021</h2>
+            <p>The NAMM Show in Anaheim, California</p>
+          </div>
+          <div class="Second_photo pic">
+            <h2>2022</h2>
+            <p>The NAMM Show Musikmesse Frankfurt, Germany</p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <section class="partners">
+        <h2 class="partners_title">Partners</h2>
+        <span class="line"></span>
+        <div class="partner_logos">
+          <a href="https://www.apple.com/apple-music/" target="_blank"
+            ><img src="./assets/Apple-Music.png" alt="Apple Music Logo"
+          /></a>
+          <a href="https://www.fender.com/en/start" target="_blank"
+            ><img src="./assets/Fender.png" alt="Fender Logo"
+          /></a>
+          <a href="http://shop.ibanez.com/" target="_blank"
+            ><img src="./assets/Ibanez.png" alt="Ibanez Logo"
+          /></a>
+          <a href="https://www.roland.com/global/" target="_blank"
+            ><img src="./assets/Roland.png" alt="Roland Logo"
+          /></a>
+          <a href="https://www.yamaha.com/en/" target="_blank"
+            ><img src="./assets/Yamaha.png" alt="Yamaha Logo"
+          /></a>
+        </div>
+      </section>
+      <div class="footer_logos footer-about-bg">
+        <img src="./assets/NAMMlogo-2.jpg" alt="Logo" />
+        <p class="footer_text">
+          © 2023 The NAMM Show(East Africa) <br />
+          All Rights Reserved
+        </p>
+      </div>
+    </footer>
```

Comment:
- [OPTIONAL] Great job implementing all the suggestions from the previous reviewer. However, I wouldn't have marked this as optional has the previous reviewer pointed it out to you that the partner section is not part of the about section on the deskktop version. As you can see, after changing the footer background color to the dark color as suggested by the previous reviewer and which is inline with the guiding template, you can see that the look of the part is totally different from what we have on the design template. I would recommend you to remove the `partner` section on the about page footer section and see how perfect your project would look in line with the design template. Please, take a look.

### Your Design
[your-about-footer-page.webm](https://user-images.githubusercontent.com/26597432/217941828-d5780664-246a-4f46-a337-a76d000c916e.webm)

### Guiding Design
![temp-about-page](https://user-images.githubusercontent.com/26597432/217941862-c72d8539-ac76-4bec-8535-35bdcc460043.png)

Reply:
-oh ok, thanks for your suggestion I'll definitely do that. 🙏...